retrievalmarket/impl/providerstates/provider_fsm.go

```diff
@@ -47,52 +45,46 @@ var ProviderEvents = fsm.Events{
 	// receiving blocks
 	fsm.Event(rm.ProviderEventBlockSent).
 		FromMany(rm.DealStatusOngoing).ToNoChange().
-		From(rm.DealStatusUnsealed).To(rm.DealStatusOngoing).
-		Action(func(deal *rm.ProviderDealState, totalSent uint64) error {
-			deal.TotalSent = totalSent
-			return nil
-		}),
-	fsm.Event(rm.ProviderEventBlocksCompleted).
-		FromMany(rm.DealStatusOngoing).To(rm.DealStatusBlocksComplete),
+		From(rm.DealStatusUnsealed).To(rm.DealStatusOngoing),
 
 	// request payment
 	fsm.Event(rm.ProviderEventPaymentRequested).
 		FromMany(rm.DealStatusOngoing, rm.DealStatusUnsealed).To(rm.DealStatusFundsNeeded).
 		From(rm.DealStatusFundsNeeded).ToJustRecord().
-		From(rm.DealStatusBlocksComplete).To(rm.DealStatusFundsNeededLastPayment).
-		From(rm.DealStatusNew).To(rm.DealStatusFundsNeededUnseal).
-		Action(func(deal *rm.ProviderDealState, totalSent uint64) error {
```

Comment:
there's two `ProviderEventPaymentRequested` events in retrievalmarket/impl/requestvalidation/revalidator.go with arguments

Reply:
this file no longer exists